generator/lib/ArtifactProviders/ExecutableSeed/babel-plugin-transform-template.ts

```diff
@@ -106,149 +118,105 @@ export function babelPluginTransformTemplate(
-                        Babel.types.identifier('schemaPath'), // There will be a global defined with this name via the `ExportNamedDeclaration` visitor
                         Babel.types.objectExpression([
                           Babel.types.objectProperty(
-                            Babel.types.identifier(datasourceUrlEnvironmentVariableName),
-                            Babel.types.identifier(`process.env.${datasourceUrlEnvironmentVariableName}`)
+                            Babel.types.identifier(`useDataProxy`),
+                            Babel.types.booleanLiteral(true)
                           ),
-                        ]),
-                        Babel.types.objectExpression([
                           Babel.types.objectProperty(
-                            Babel.types.identifier('forcePrismaLibrary'),
-                            Babel.types.booleanLiteral(true)
+                            Babel.types.identifier(`connectionString`),
+                            Babel.types.identifier(`process.env.${datasourceUrlEnvironmentVariableName}`)
                           ),
                           Babel.types.objectProperty(
-                            Babel.types.identifier('resolve'),
+                            Babel.types.identifier('schema'),
                             Babel.types.objectExpression([
                               Babel.types.objectProperty(
-                                Babel.types.stringLiteral('.prisma/client'),
-                                Babel.types.callExpression(
-                                  Babel.types.memberExpression(
-                                    Babel.types.identifier('path'),
-                                    Babel.types.identifier('join')
-                                  ),
-                                  [
-                                    Babel.types.callExpression(
-                                      Babel.types.memberExpression(
-                                        Babel.types.identifier('process'),
-                                        Babel.types.identifier('cwd')
-                                      ),
-                                      []
-                                    ),
-                                    Babel.types.stringLiteral('node_modules/.prisma/client/index.js'),
-                                  ]
+                                Babel.types.identifier('contents'),
+                                Babel.types.identifier(
+                                  `process.env.${transformedPrismaSchemaContentEnvironmentVariableName}`
                                 )
                               ),
+                              Babel.types.objectProperty(
+                                Babel.types.identifier('path'),
+                                Babel.types.identifier(`"${schemaPath}"`)
```

Comment:
The schema path is hardcoded to `/tmp/schema.prisma` however we could improve this by writing it in`tmp/[environmentId]/schema.prisma` . However I think it would have to be passed in through `process.env` from PDP. I'm not sure how to pass a string argument with execa without nesting inside the `env` property